contrib/generate_precompile.jl

```diff
@@ -347,8 +347,9 @@ generate_precompile_statements() = try # Make sure `ansi_enablecursor` is printe
             Base.Precompilation.precompilepkgs(;fancyprint=true);
             $precompile_script
             """
+        cpu_target = unsafe_string(Base.JLOptions().cpu_target)
         p = run(pipeline(addenv(`$(julia_exepath()) -O0 --trace-compile=$tmp_proc --sysimage $sysimg
-                --cpu-target=native --startup-file=no --color=yes --project=$(pkgpath)`, procenv),
+                --cpu-target=$cpu_target --startup-file=no --color=yes --project=$(pkgpath)`, procenv),
```

Comment:
Isn't this process just creating precompile statements? I think `--cpu-target=native` and `-O0` is specified to speed it up?

Reply:
Oh I see. Yeah this isn't needed then, sorry for the noise.